pybsn/__init__.py

```diff
@@ -26,38 +26,138 @@
     UNICODE_TYPE = None
 
 class Node(object):
+    """ Higher level "Node" abstraction for PyBSN.
+
+    In this abstraction, the BigDB tree is represented as dynamically created nodes. You can
+    navigate through the tree by traversing the child nodes of the root object.
+
+    Hyphens (-) in the REST API are converted to _ for python.
+
+    E.g.,
+      root.core.switch references the node controller/core/switch
+      root.core.switch_config references the node controller/core/switch-config
+
+    On a node, you can call HTTP methods, e.g.,
+
+    root.core.switch_config.post(switch_data)
```

Comment:
This write example might be something we might want to copy over to the README.md.

I've initially tried out things like `root.foo.bar = {'a': 42}` in repl, searching for ways to request writing things, since reading values and schema were like `()`, `#`

Reply:
#28

Reply:
#28